crates/hyperswitch_interfaces/src/lib.rs

```diff
@@ -1,7 +1,17 @@
 //! Hyperswitch interface
 
-#![warn(missing_docs, missing_debug_implementations)]
+pub mod api;
 
-pub mod secrets_interface;
+pub mod configs;
 
 pub mod encryption_interface;
+
+pub mod errors;
+
+pub mod events;
+
+pub mod metrics;
+
+pub mod secrets_interface;
+
+pub mod types;
```

Comment:
Also, you can remove the newlines between module declarations.

Reply:
Done